index.rst

```diff
@@ -0,0 +1,42 @@
+Dask Examples
+=============
+
+These examples show how to use Dask in a variety of situations.
+
+First, there are some high level examples about various Dask APIs like arrays,
+dataframes, and futures, then there are more in-depth examples about
+particular features or use cases.
+
+You can run these examples in a live session here: |Binder|
+
+.. |Binder| image:: https://mybinder.org/badge.svg
+   :target: https://mybinder.org/v2/gh/dask/dask-examples/master?urlpath=lab
+
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Basic Examples:
+
+   array
+   dataframe
+   delayed
+   futures
+   machine-learning
```

Comment:
What about embarrassingly-parallel example 😢 

Joking aside, I can understand it's difficult to put it somewhere in this page.